operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Entries should be ordered as follows:
 Entries should include a reference to the Pull Request that introduced the change.
 
 ## main / unreleased
+* [ENHANCEMENT] Add extra values for KEDA auto scaling. #10265
```

Comment:
can you clarify what the extra values are? We're trying to have the changelog be sufficient for everyone to understand what the change is and to even start using it. Here I thikn it's sufficeint to mention what new options are supported (e.g. `authenticaiton` to KEDA, ignoring null values, etc.)